HPXMLtoOpenStudio/resources/model.rb

```diff
@@ -804,6 +821,21 @@ def self.merge_unit_models(model, hpxml_osm_map)
         end
       end
     end
+    return if unit_surface_to_obj_index_map.empty?
+
+    model_objects.each_with_index do |obj, index|
+      next unless unit_surface_to_obj_index_map.values.include? index
+
+      unit_surface_handle = unit_model_objects[index].handle
+      surface = obj.to_Surface.get
+      adjacent_surface_index = unit_surface_to_obj_index_map[unit_model_map[unit_surface_handle]]
+      adjacent_surface = model_objects[adjacent_surface_index].to_Surface.get
+      next if surface.adjacentSurface.is_initialized
+
+      surface.setAdjacentSurface(adjacent_surface)
+      # Need to set the same construction to make OS working
+      adjacent_surface.setConstruction(surface.construction.get)
```

Comment:
I forget, did we discuss a plan for ensuring constructions are identical for the two walls that point to each other?

For shared boilers, I was proposing that one would describe the shared boiler once (in the first dwelling unit attached to it, e.g. [here](https://github.com/NREL/OpenStudio-HPXML/blob/c074c73f89f88dab8ecb52230098dd947648c69e/workflow/sample_files/base-bldgtype-mf-whole-building-shared-boilers-sequenced-operation.xml#L349-L362)), and then every other dwelling unit would have a boiler with the `sameas` attribute where no additional details would be provided (e.g., [here](https://github.com/NREL/OpenStudio-HPXML/blob/c074c73f89f88dab8ecb52230098dd947648c69e/workflow/sample_files/base-bldgtype-mf-whole-building-shared-boilers-sequenced-operation.xml#L878-L881)) so as to prevent inconsistency. If we went that path, ideally we could enforce it via the Schematron validation. Is that the path you're going to go? Or did you have another idea in mind?

Reply:
Yes that's what I think, add more error-checking in EPvalidator to make sure there's no inconsistent construction, did you disallow the boiler inputs when using sameas? I can do the same restriction or I can check the constructions are exactly the same.

Reply:
I don’t remember if I got that far with the shared boiler implementation, but I would favor disallowing the inputs.

Reply:
Checked the links, it seems that you disallowed additional inputs, I'm going to do the same.